cmd/run.go

```diff
@@ -372,3 +379,18 @@ func init() {
 
 	RootCmd.AddCommand(RunCmd)
 }
+
+// GetFreePort asks the kernel for a free open port that is ready to use.
+func getFreePort() (int, error) {
```

Comment:
@jigargandhi we are already importing `"github.com/phayes/freeport"` in the `pkg/standalone/run.go` file. Why can't we use that here?